src/Symfony/Bridge/Twig/Form/TwigRenderer.php

```diff
@@ -19,23 +19,26 @@
  */
 class TwigRenderer extends FormRenderer implements TwigRendererInterface
 {
-    /**
-     * @var TwigRendererEngineInterface
-     */
-    private $engine;
-
     public function __construct(TwigRendererEngineInterface $engine, $csrfTokenManager = null)
     {
         parent::__construct($engine, $csrfTokenManager);
+    }
 
-        $this->engine = $engine;
+    /**
+     * Returns the engine used by this renderer.
+     *
+     * @return TwigRendererEngineInterface The renderer engine
```

Comment:
this is added so that static code analysis and autocomplete works (which I assume is the also the only reason the private engine property was overwritten)